src/python/gudhi/nerve_gic.pyx

```diff
@@ -1,15 +1,20 @@
-# This file is part of the Gudhi Library - https://gudhi.inria.fr/ -
-# which is released under MIT.
-# See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full
-# license details.
+from __future__ import print_function
```

Comment:
Why does this specific import end up in a different place than the others? Is it the result of some automatic tool?
(We should get rid of it (so its position does not matter) since we do not support python2 anymore, but not this PR)

Reply:
I removed all the `from __future__ import print_function`

Reply:
This file will disappear anyway, but just as information:

> Python mandates that future-imports must appear in the module before any other code except docstrings.

That is why this line got separated from the rest, even though it could have probably have been put below the comment (and before the __licence__ etc.).
PEP recommandations are: first docstring, then future-imports, then any metadata dunders, then any imports in order of global to local.

Reply:
Thanks, that's exactly what I was missing :smiley: